openedx/core/djangoapps/schedules/models.py

```diff
@@ -6,6 +6,12 @@
 from config_models.models import ConfigurationModel
 
 
+EXPERIENCES = (
+    (0, 'Recurring Nudge and Upgrade Reminder'),
+    (1, 'Course Updates and Upgrade Reminder'),
```

Comment:
Given that in the "0" case the two things described are both separate communications, I think it might make sense to just make this "Course Updates" since the upgrade reminder is embedded within that communication.